app/uk/gov/hmrc/eoricommoncomponent/frontend/views/subscription/sub01_outcome_processing.scala.html

```diff
@@ -18,33 +18,25 @@
 @import uk.gov.hmrc.eoricommoncomponent.frontend.views.ServiceName._
 @import uk.gov.hmrc.eoricommoncomponent.frontend.views.helpers.DateFormatter
 @import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.eoricommoncomponent.frontend.viewModels.Sub01OutcomeViewModel.headingForProcessing
 
-@this(layout_di: layout, feedback_di: helpers.feedback, dateFormatter: DateFormatter, govukPanel : GovukPanel)
+@this(layout_di: layout, feedback_di: helpers.feedback, dateFormatter: DateFormatter, govukPanel: GovukPanel, h2: helpers.h2, p: helpers.paragraph)
 
 @(orgName: Option[String], processedDate: String)(implicit messages: Messages, request: Request[_])
 
-@heading = {@orgName match {
-    case Some(org) => {@messages("cds.sub01.outcome.processing.heading", shortName, org)}
-    case None => {@messages("cds.sub01.outcome.processing.heading-noname", shortName)}
-  }
-}
-
 @confirmationHtml = {
     <p id="processed-date" class="govuk-!-font-size-36"><strong>@messages("cds.subscription.outcomes.in-processing.received", dateFormatter.format(processedDate))</strong></p>
 }
 @layout_di(messages("cds.sub01.outcome.processing.title", shortName), displayBackLink = false) {
-
     <div>
         @govukPanel(Panel(
-            title = HtmlContent(heading),
+            title = HtmlContent(headingForProcessing(orgName)),
             content = HtmlContent(confirmationHtml)
         ))
-
         <div id="what-happens-next">
-            <h2 class="govuk-heading-m">@messages("cds.subscription.outcomes.steps.next")</h2>
-            <p class="govuk-body">
-                @messages("cds.sub01.outcome.processing.we-are-processing", shortName)
-            </p>
+            @h2(messages("cds.subscription.outcomes.steps.next"))
+            @p(Some("processing-para"), messages("cds.sub01.outcome.processing.we-are-processing", shortName))
```

Comment:
Following discussion we had earlier, the ID is going to be the second param now to make it optional. You might have the same issues as the other ticket if this isn't updated? therefore might be worth locally switching the params in the 'paragraph' helper and changing the @p tag here to be `@p(messages("cds.sub01.outcome.processing.we-are-processing", shortName), Some("processing-para"))`